doczrc.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  title: 'IDYEAHS',
+  description: 'Because ideas are the currency of the future',
+  theme: 'idyeahs-theme',
+  themeConfig: {
+    colors: {
+      primary: 'rebeccapurple',
```

Comment:
Wenn du hier `rebeccapurple` als primary color definierst, kannst du es dann nicht in den folgenden Dateien über diese Variable aufrufen?

Reply:
dies ist nur für die Docu (docz, living styleguide). Aber das nehme ich noch in Angriff wenn ich das Styling überarbeite, dass ich variablen für meine Hauptfarben vergebe